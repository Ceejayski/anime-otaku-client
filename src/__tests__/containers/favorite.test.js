import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Favorites from '../../containers/favorites';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <BrowserRouter>
        <Favorites />
      </BrowserRouter>
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
