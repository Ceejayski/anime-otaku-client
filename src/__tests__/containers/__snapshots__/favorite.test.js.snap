// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div
  style={
    Object {
      "minHeight": "70vh",
    }
  }
>
  <h3>
     No Anime Added Yet! 
  </h3>
</div>
`;
